docs/Video Streaming for Navigation Apps/Displaying Turn Directions/index.md

```diff
@@ -1,13 +1,15 @@
 # Displaying Turn Directions
 While your app is navigating the user, you will also want to send turn by turn directions. This is useful for if your app is in the background or if the user is in the middle of a phone call, and gives the system additional information about the next maneuver the user must make.
 
-To display a Turn by Turn direction, a combination of the `SDLShowConstantTBT` and `SDLAlertManeuver` RPCs must be used. The `SDLShowConstantTBT` RPC involves the data that will be shown on the head unit. The main properties of this object to set are `navigationText1`, `navigationText2`, and `turnIcon`. A best practice for navigation applications is to use the `navigationText1` as the direction to give (Turn Right) and `navigationText2` to provide the distance to that direction (3 mi). When an `SDLAlertManeuver` is sent, you may also include accompanying text that you would like the head unit to speak when an direction is displayed on screen (e.g. In 3 miles turn right.).
+To display a Turn by Turn direction, a combination of the @![iOS]`SDLShowConstantTBT`!@  @![android,javaSE,javaEE] `Add Property`!@ and @![iOS]`SDLAlertManeuver`!@  @![android,javaSE,javaEE] `Add Property`!@ RPCs must be used. The @![iOS]`SDLShowConstantTBT`!@  @![android,javaSE,javaEE] `Add Property`!@ RPC involves the data that will be shown on the head unit. The main properties of this object to set are `navigationText1`, `navigationText2`, and `turnIcon`. A best practice for navigation applications is to use the `navigationText1` as the direction to give (Turn Right) and `navigationText2` to provide the distance to that direction (3 mi). When an @![iOS]`SDLAlertManeuver`!@  @![android,javaSE,javaEE] `Add Property`!@ is sent, you may also include accompanying text that you would like the head unit to speak when an direction is displayed on screen (e.g. In 3 miles turn right.).
```

Comment:
There are multiple spaces before the android/java code, and the android/java code is not added. I assume "Add Property" is a TODO?

Reply:
Updated to show TODO removed extra spaces